### DISCUSSION TITLE
Hi All

### DISCUSSION THREAD

**Discussion by havyarimanaj** [OpenMined/courses#131] [Introduce Yourself]
Hi everyone and happy 2021!
I am Jean-Claude and really glad to join this course and e-meet your all!
I work in Humanitarian Action Field:  Data, Data protection, Data privacy., Do no harm, Tech for Good, etc. are among the top priorities with push for Digital Transformation in the sector. So this course is just great and well timed .... I look forward to dive in and please if you have thoughts or uses cases in Humanitarian Field, let me know.

**Comment by yemikifouly** [comment]
Hello Jean-Claude! 🙂 Welcome to OpenMined and the discussion board! Wow! Giving your interest in data protection and privacy you are definitely at the right place. 🙂 I hope you enjoy the course very much!
